docs/release/release-team-onboarding.md

```diff
@@ -0,0 +1,60 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Onboarding Notes for Release Team Shadows](#onboarding-notes-for-release-team-shadows)
+  - [Overview](#overview)
+    - [Release Lead Shadows Team](#release-lead-shadows-team)
+    - [Comms Lead Shadows Team](#comms-lead-shadows-team)
+    - [CI Signal Lead Shadows Team](#ci-signal-lead-shadows-team)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Onboarding Notes for Release Team Shadows
+
+## Overview
+
+Welcome to the Cluster API Release Team Shadows onboarding documentation!
+
+As a member of the Release Team and Shadow, you play a crucial role in helping to ensure project releases
+are delivered smoothly and in a timely manner. While there are specific responsibilities for each sub-team within
+the Release Team Shadows, below are some general notes that every member of the Releease Team might benefit by going 
+through at the beginning of the cycle:
+
+- Slack:
+    - Most discussions related CAPI Release topics happens in the #cluster-api channel on the Kubernetes Slack. If you need access to the Kubernetes Slack, please visit http://slack.k8s.io/.
+- Kubernetes SIG membership:
+    -  Try to become an official member of the Kubernetes SIG, if possible. More information on the membership and requirements can be found [here](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-team.md#cluster-api-release-team-vs-kuberneteskubernetes-sig-membership).
+- Familiarize yourself with the Release Process:
+    - Review the release [tasks document](../release/release-tasks.md) which explains the responsibilities and tasks for each role within the release team.
+- Check the Release Timeline:
+    - Go through the [release timeline](../release/releases) of the release cycle you are involved in (i.e checkout `release-1.6.md` if you are part of the 1.6 cycle release team) to better understand the key milestones and deadlines.
+
+Now, let's dive into the specific onboarding notes for each sub-team below.
+
+### Release Lead Shadows Team
+
+- Explore CI Setup and Tools:
+    - Gain some insights into how CI is setup in the Cluster API and tools used. You can check out CI Team onboarding [notes](#ci-signal-lead-shadows-team) for more details.
+- Learn about Release Notes Generation:
+    - Get to know how the release notes generation tools works.
+- Review the onboarding notes for both the Comms and CI Lead Shadows teams:
+    - Check out the onboarding notes of [Comms](#comms-lead-shadows-team) and [CI Signal](#ci-signal-lead-shadows-team) teams.
+
+### Comms Lead Shadows Team
```

Comment:
@g-gaston i would be happy to know your opinion on this one and add anything I missed here